src/relay/transforms/memory_alloc.cc

```diff
@@ -91,9 +91,15 @@ class DialectRewriter : public transform::DeviceAwareExprMutator {
 
     for (auto field : tuple_node->fields) {
       auto new_field = Mutate(field);
-      if (new_field->IsInstance<ConstantNode>()) {
-        Var const_var("const", Type(nullptr));
-        new_field = scope.Push(const_var, new_field);
+      if (auto op = new_field.as<ConstantNode>()) {
```

Comment:
nit: const auto* op
Seems to be the idiom (and matches my personal style :-)

Reply:
done